Lib/idlelib/configdialog.py

```diff
@@ -570,9 +571,14 @@ def create_page_font_tab(self):
         indent_title = Label(
                 frame_indent, justify=LEFT,
                 text='Python Standard: 4 Spaces!')
-        self.indent_scale = Scale(
-                frame_indent, variable=self.space_num,
-                orient='horizontal', tickinterval=2, from_=2, to=16)
+        try:
+            self.indent_chooser = Spinbox(
+                    frame_indent, textvariable=self.space_num,
+                    state="readonly", from_=2, to=16)
```

Comment:
I want to allow 1.  I think max 10 is enough.  Will ask on idle-dev.  If 16 were actually needed, I would want to allow key entry in entry box.  Will want key entry if use for font size.
Needs width=2, at least when moved.